shared/css/phase1-design-system.scss

```diff
@@ -0,0 +1,387 @@
+@import 'color';
+@import 'barlowSemiCondensed-font';
+@import 'font';
```

Comment:
Nit: these can all be imported on one line:
`@import "color.scss", "font.scss", "barlowSemiCondensed-font";`

Reply:
True. If it's critical, I can update it.
However I see that imports are currently organized in the way it's organized here, so I guess it'll be better to leave it as it is. Plus, to me it looks easier to read)

Reply:
Not critical, it's fine to leave it as-is 👍